src/internal/dbutil/testdb.go

```diff
@@ -1,79 +0,0 @@
-package dbutil
```

Comment:
I think this is gone in master? I'm not really sure what's going on with this diff.